### PR TITLE
D2K - Add Harkonnen Mission 2a

### PR DESCRIPTION
**Pull request by MustaphaTR**

### CODE REVIEW

File: mods/d2k/maps/harkonnen-02a/map.yaml [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+		LockColor: True
+		Color: FE0000
+		Enemies: Atreides, Creeps
+	PlayerReference@Atreides:
+		Name: Atreides
+		LockFaction: True
+		Faction: atreides
+		LockColor: True
+		Color: 9191FF
+		Enemies: Harkonnen, Creeps
```
Comment (abcdefg30) [inline mods/d2k/maps/harkonnen-02a/map.yaml]: I think we should remove `Creeps` from that list, Atreides reinforcements often just went after the worm directly after being dropped off by the carryall in my tests.

**Review by abcdefg30 — APPROVED** [review]
Looks good to me aside from that one nit. 👍 / ✅

File: mods/d2k/maps/harkonnen-02a/harkonnen02a.lua [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+	end)
+end
+
+IdleHunt = function(unit)
+	Trigger.OnIdle(unit, unit.Hunt)
+end
+
+Tick = function()
+	if player.HasNoRequiredUnits() then
+		atreides.MarkCompletedObjective(KillAtreides)
```
Comment (ltem) [inline mods/d2k/maps/harkonnen-02a/harkonnen02a.lua]: Shouldn't this be `atreides.MarkCompletedObjective(KillHarkonnen)` ?

**Comment by MustaphaTR** [comment]
Updated.

**Comment by ltem** [comment]
:+1:

**Review by ltem — APPROVED** [review]

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/93be590547fb0d37ea6e669086fa209e4fef7c8a...a1936bf5bc76342a4ed4a5229356f14c50fc6004)

**Pull request merged**